src/SDK/Common/Configuration/Configuration.php

```diff
@@ -46,9 +53,4 @@ public static function getRatio(string $key, float $default = null): float
     {
         return Accessor::getRatio($key, $default ? (string) $default : null);
```

Comment:
`Configuration::getRatio('not-set', 0)` should not throw.

Reply:
I might have fixed this in a recent commit, it's behaving as it should now and returning `0.0`; I've added a test for it.